### PR TITLE
Add a pre-check for E2E tests

### PR DESCRIPTION
**Pull request by fgksgf**
### Type of change:

- [x] CI

### What this PR does / why we need it:

In #949 we use `ginkgo --focus` to run test cases in parallel, but some newly added cases may not follow the naming convention. So we need to add a pre-check to avoid some E2E cases not running.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### Test

Tested on Linux:
image

### CODE REVIEW

**Comment by fgksgf** [comment]
@tao12345666333 plz add a reviewer so that the e2e CI job can run.

**Comment by fgksgf** [comment]
The script can find we missed the prefix in some files:
image

**Comment by tao12345666333** [comment]
> The script can find we missed the prefix in some files

Great!!

**Comment by tao12345666333** [comment]
re-run CI.

**Review by tao12345666333 — COMMENTED** [review]
Should we add a `check-e2e-name` rule to the Makefile? It might be more convenient

**Comment by fgksgf** [comment]
> Should we add a `check-e2e-name` rule to the Makefile? It might be more convenient

Good idea

**Review by tao12345666333 — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -203,3 +204,8 @@ update-gofmt:
 ### update-all:           Update all update- rules.
 .PHONY: update-all
 update-all: update-codegen update-license update-mdlint update-gofmt
+
+### names-check:          Check if e2e test cases' names have the prefix "suite-<suite-name>".
+.PHONY: names-check
```
Comment (lingsamuel) [inline Makefile:209]: I think `e2e-names-check` is better because it explicitly indicates that it is related to the e2e test.

**Review by lingsamuel — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (fgksgf) [reply Makefile:209]: ok, i'll fix it

**Pull request merged**